src/xunit.performance.api/PerformanceTestConfig.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    public class PerformanceTestConfig
+    {
+        public PerformanceTestConfig() 
+        {
+            Iterations = 30;
```

Comment:
Do we want default values here? Should these be passed in the constructor instead?